io.github.markummitchell.Engauge_Digitizer.json

```diff
@@ -0,0 +1,132 @@
+{
+	"app-id": "io.github.markummitchell.Engauge_Digitizer",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.10",
+	"sdk": "org.kde.Sdk",
+	"command": "engauge",
+	"rename-icon": "engauge-digitizer",
+	"rename-desktop-file": "engauge-digitizer.desktop",
+	"rename-appdata-file": "engauge-digitizer.appdata.xml",
+	"finish-args": [
+		"--socket=wayland",
+		"--socket=x11",
+		"--share=ipc",
+		"--device=dri"
+	],
+	"cleanup": [
+		"/include",
+		"/lib/pkgconfig",
+		"/lib/cmake",
+		"/share/aclocal",
+		"/share/pkgconfig",
+		"*.a",
+		"*.la",
+		"*.cmake"
+	],
+	"modules": [
+		{
+			"name": "log4cpp",
```

Comment:
In order to fix the build on aarch64 you need to rm-configure: true and then add an autoreconf autogen.sh as per this example: https://github.com/flathub/com.tux4kids.tuxmath/blob/cce3487f0c53322983a9cefd531de783241d96f8/com.tux4kids.tuxmath.json#L32

Reply:
Thanks, done

Reply:
Could you link the PR here for future reference.

Reply:
https://github.com/markummitchell/engauge-digitizer/pull/255